src/Model/User.php

```diff
@@ -99,6 +99,20 @@ public static function getById($uid)
 		return DBA::selectFirst('user', [], ['uid' => $uid]);
 	}
 
+	/**
+	 * @param  integer       $uid
+	 * @return array|boolean User record if it exists, false otherwise
```

Comment:
Does it return a `User` record or a public Contact Id?

Reply:
It does return some contact stuff - but the input value is the user id. I had been unsure where to place it. In the Contact class it had to named differently as well, I think.

Reply:
It probably should by in Contact as `getPublicIdByUserId` and have an `int` return value.

Reply:
Moved